### ISSUE TITLE
Formatting

### ISSUE THREAD

**Issue by Paxa** [yamadapc/pyjamas#16]
On mac I see it as: (I see that structurized list for a short moment then it disappear)

```
% dub --config=test                                        
Target colorize 1.0.5 is up to date. Use --force to rebuild.
Target bed 0.0.5 is up to date. Use --force to rebuild.
Target pyjamas 0.2.1+commit.2.g4172488 is up to date. Use --force to rebuild.
Running ./test 
                                ✓ asserts whether a range is sorted          ✓ asserts whether a value is bigger than other          ✓ asserts whether a value is smaller than other 27 passing
with `opIndex` definedty for arraysfalse

```

![screen shot 2015-01-10 at 16 36 58](https://cloud.githubusercontent.com/assets/26019/5691070/f7c2056c-98e6-11e4-9469-764030982e48.png)

Is it feature or bug? I compare with travis output, it's different...

Do you think it will be useful to add command line options for format, like in rspec?
https://relishapp.com/rspec/rspec-core/v/2-6/docs/command-line/format-option

**Comment by yamadapc** [comment]
Yes! This is from `bed` and if I'm not mistaken it does support different "reporters" though its not implemented in a polished way. I'm going to look into this problem when I have some time. It's from `bed`, which first writes that tests are pending and later moves to the pending markers and updates them (for concurrently running tests - supposedly, as I haven't worked on that feature for a while; it's on the `dev` branch). There's probably some error in the escape sequences and what nots. Thanks for reporting.

**Comment by yamadapc** [comment]
But this belongs [here](github.com/yamadapc/bed) though.
